datastream-bigquery/src/main/java/com/linkedin/datastream/bigquery/BigqueryBatchCommitter.java

```diff
@@ -154,7 +197,7 @@ public void commit(List<InsertAllRequest.RowToInsert> batch,
             InsertAllResponse response = null;
 
             try {
-                createTableIfAbsent(destination);
+                createOrUpdateTable(destination);
```

Comment:
I understand that we compare schema at 2 places: 1) adding each record in a batch and 2) commit current batch to GBQ. What if we update schema only when getting exception from commit a batch? Just a quick idea, we can discuss this at some time.